tables/isolates.csv

```diff
@@ -1302,3 +1302,10 @@ B.1.640.2 Spike,B.1.640.2,FALSE,7314471,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K040015/2022,Omicron/BA.4,FALSE,12268495,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K040013/2022,Omicron/BA.5,FALSE,12268493,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K032888/2021,Omicron/BA.1,FALSE,7886688,NULL,NULL,TRUE
+BA.1 Spike:+376A+405N+408S+452R-446S+486V-493R-496S-547K-856K-981F,Omicron/BA.1,FALSE,NULL,NULL,NULL,TRUE
+SARS-CoV-2/human/BEL/rega-20174/2021,Omicron/BA.1,FALSE,NULL,OL672836,NULL,TRUE
+B.1.351 full genome:-80A-246I,Beta,FALSE,NULL,NULL,NULL,TRUE
+B.1.617.2 full genome:142D+682Q,Delta,FALSE,NULL,NULL,NULL,TRUE
+Omicron/BA.2 full genome:-27del+27S-142D-440K+683W,Omicron/BA.2,FALSE,NULL,NULL,NULL,TRUE
+Omicron/BA.4 full genome,Omicron/BA.4,FALSE,NULL,NULL,NULL,TRUE
+Omicron/BA.4 full genome:3G-658S,Omicron/BA.4,FALSE,NULL,NULL,NULL,TRUE

```

Comment:
`Omicron/BA.4 full genome` and `micron/BA.4 full genome:3G-658S` here are based on the assumption that [hCoV-19/South Africa/CERI-KRISP-K040015/2022](https://github.com/hivdb/covid-drdb-payload/blob/master/tables/isolates.csv#L1302) represents the consensus `Omicron/BA.4 full genome`